project/http/requests/useragent/userAgent.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+class UserAgent:
```

Comment:
Just for clarify lets rename class to something like UserAgentManager or UserAgentHandler